Dockerfile.mysql

```diff
@@ -0,0 +1,11 @@
+ARG MYSQL_VERSION=8.0.37-bookworm
+FROM mysql:${MYSQL_VERSION}
+
+ENV MYSQL_DATABASE=app_db
+ENV MYSQL_USER=app_user
+ENV MYSQL_PASSWORD=1234
+ENV MYSQL_ROOT_PASSWORD=1234
+
+VOLUME /var/lib/mysql
+
+EXPOSE 3306
```

Comment:
Add Eof(End-of-file), need to add empty line at the end
![image](https://github.com/mate-academy/devops_todolist_docker_core_task_2_volumes/assets/87720583/e3d02eaf-2941-43ce-8115-85314fa9fde2)